src/trace.js

```diff
@@ -34,13 +34,14 @@ export default {
 
   id: 'crosshair',
 
-  afterInit: function(chart) {
+  start: function(chart) {
     
-    if (!chart.config.options.scales.x) {
-      return
+    const xScaleOptions = chart.config.options.scales.x || chart.config.options.scales.xAxes;
```

Comment:
Would be better to make this a variable that the user can configure in the options for the plugin so that all names you give to your x axis are supported. 

Then if no name has been given you can fall back to chart.js's default x axis scale id which is `x`

Reply:
You are, of course, welcome to make any changes you desire.